res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+			<speed min="40" max="80" />
+			<production_time value="60" />
+			<programme_data_modifiers>
+				<modifier name="topicality::age" value="5" />
+				<modifier name="topicality::refresh" value="2" />
+			</programme_data_modifiers>
+		</scripttemplate>
+
+		<scripttemplate product="3" licence_type="1" guid="f13dd881-b149-4981-aa75-67dd6fe9663a" creator="9551" comment="TODO: I am not happy with the titles and descriptions of these.">
+			<title>
+				<de>%tower% %latebreaking%: %title%</de>
+				<en>%tower% %latebreaking%: %title%</en>
+			</title>
+			<description>
+				<de>Die neuesten und heißesten Videospiele werden hier präsentiert und getestet. Nur für euch! Außerdem könnt ihr im Studio gegeneinander antreten und Preise gewinnen!</de>
+				<en>The latest and hottest video games will be presented and tested here. Just for you! You can also compete against each other in the studio and win prizes!</en>
+			</description>
+
+			<variables>
+				<tower>
+					<de>TVT|Tower</de>
+					<en>TVT|Tower</en>
+				</tower>
+				<latebreaking>
+					<de>Brandaktuell|Aktuell</de>
+					<en>Late-Breaking|Hot</en>
+				</latebreaking>
+				<title>
+					<de>Die Videospiele-Show|Arcade Action|Hot Games|Video Action</de>
+					<en>The Videogame Show|Arcade Action|Hot Games|Video Action</en>
+				</title>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="64" required="0" />
+			</jobs>
+			<data flags="1" production_limit="9000" production_broadcast_limit="2" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="104" subgenres="2,16" />
+			<groups target_groups_optional="19" />
+			<blocks value="1" />
+			<price min="10000" max="20000" />
+			<potential min="40" max="80" />
+			<outcome min="40" max="80" />
+			<review min="40" max="80" />
+			<speed min="40" max="80" />
+			<production_time value="60" />
+			<programme_data_modifiers>
+				<modifier name="topicality::age" value="5" />
+				<modifier name="topicality::refresh" value="2" />
+			</programme_data_modifiers>
+			<targetgroupattractivity children_male="2.0" children_female="0" teenagers_male="2.0" teenagers_female="0" housewives_male="0" housewives_female="-1.0" employees_male="1.0" employees_female="-0.5" unemployed_male="2.0" unemployed_female="0" managers_male="0.5" managers_female="0" pensioners_male="-1.0" pensioners_female="-2.0" comment="TODO: Does this work in scripts? Clarify." />
+		</scripttemplate>
+
+		<scripttemplate product="4" licence_type="1" guid="14c23a4a-9d4b-4c87-8890-45652ce81cd4" creator="9551" comment="inspired from programme from original MadTV">
+			<title>
+				<de>Die Götterwelt von %bali%</de>
+				<en>The Gods of %bali%</en>
+			</title>
+			<description>
+				<de>Alle 412.000 Götter werden vorgestellt.
```

Comment:
The variable should be a nested one:
country_code: 1|2|3
country_1: <de>Bali</de>
country_2: <de>...</de>
gods_1: <de>412.000</de>
gods_2: <de>...</de>

Just to avoid that god amount "mixes" with each creation (for the same country)

Reply:
@GWRon can you give me an XML example of what you mean?

Reply:
I don't really think that each country needs a unique number. And it was meant only as a suggestion.
Wrt. to the syntax, the comment already gives you the basic idea. The culture scripts in Ronny's user file contain examples of such choices. (One variable selects the variable name suffix which is then used in other places to put together the variable names.)

Reply:
my user file does not contain such an example.
The idea is to forge the variables in two steps:
```xml
<variables>
  <base>
    <de>1|2|3</de>
  </base>
  <extended>
    <de>Die ${count_${base}} Götter von ${country_${base}}</de>
  </extended>
  <count_1>
    <de>2|5</de>
  </count_1>
  <count_2>
    <de>412.000|365.000|123.000</de>
  </count_2>
  <count_3>
    <de>5|12|34</de>
  </count_3>
  <country_1>
    <de>Bali</de>
  </country_1>
  <country_2>
    <de>Helogland</de>
  </country_2>
  <country_3>
    <de>Mordor</de>
  </country_3>
</variables>
```
The variable resolver will resolve "inner" first - and so settle for 1 _or_ 2 _or_ 3 and it will be the same for all upcoming variables in the programme which use this variable.
In the next step the evaluation will see ${count_1} (or 2 or 3) and the very same number also for the country variable.

Reply:
Thanks. Applying that format.

Without the variable number of gods though, as it the number is part of the joke and fixed/dependent on `production_limit` (which will be changed to 100 as requested further above). *Unless* I can specify a complex script for `production_limit` too. `;)`

Reply:
If you don't have variable content that depends on other variables' choices, you don't need this format.